servers/quarkus-server/src/test/java/org/projectnessie/server/TestRest.java

```diff
@@ -119,10 +121,39 @@ void closeClient() {
   }
 
   @Test
-  void illegalArgumentException() {
-    assertThat(
-        assertThrows(NessieBadRequestException.class, () -> tree.createReference(Hash.of("12341234cafebeef"))).getMessage(),
-        startsWith("Bad Request (HTTP/400): Only tag and branch references can be created"));
+  void createReferences() throws NessieNotFoundException {
```

Comment:
is it a bad diff render on github or has the `illegalArgumentException` test been removed. If yes then how come?

Reply:
Not gone, just integrated in that new test method.